### PR TITLE
Restructure to add context

### PR DESCRIPTION
**Pull request by cdb1212**
Restructure so templates and classes are in the user dash board, managing sessions is in the class dashboard, all pages have context, and nav buttons don't redirect when disabled. Solves everything in restructure issue except headers, and will need to uncomment the route for ClassDashboard to integrate with previous pull request

### CODE REVIEW

**Comment by logand** [comment]
Before I can check this out, I need to get the app up and running. Please redo your readme to include instructions on how to run the application and remove the readme from the create react app repo.

**Comment by cdb1212** [comment]
remove border top on tabs, change color of background to match tab

File: src/components/CreateCollectionView.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React from "react";
 import List from "./List";
 import Alerts from "./Alerts";
 import * as db from "../util/dbfunctions";
-import PlanFunctionBar from "./PlanFunctionBar";
+import CreateFunctionBar from "./CreateFunctionBar";
 
-class PlanSessionView extends React.Component {
+class CreateCollectionView extends React.Component {
```
Comment (logand) [inline src/components/CreateCollectionView.js]: Here you have refactored to call them collections but I believe you still call them sessions and templates in the app. Why the name discrepancy?

Comment (cdb1212) [reply src/components/CreateCollectionView.js]: Collection was the name I was using on the backend to reference both Sessions and Templates, as they are both a type of collection I guess. The function of that page now works for both Sessions and Templates, so I felt the previous name was now not applicable

**Pull request merged**